pkg/openservicebroker/server/apiserver.go

```diff
@@ -0,0 +1,73 @@
+package server
```

Comment:
@deads2k I think this should be under pkg/template/servicebroker - do you agree; will you change or should I in a subsequent commit?